examples/map_transformations/reprojection_different_observers.py

```diff
@@ -29,16 +29,17 @@
 plt.rcParams['figure.figsize'] = (16, 8)
 
 ######################################################################
-# Let’s download an EUV image from both AIA and STEREO A, when their
-# separation was around 90 degrees.
+# Let’s download an EUV image from both AIA and STEREO A
```

Comment:
Do we not want `# separation was around 90 degrees.`?

Reply:
It isn't any more, because I changed the images so it's more like 120 degrees (off the top of my head). Can edit it to 120 deg instead?

Reply:
If you know what it is sure, if not leave it.